src/utilities.jl

```diff
@@ -504,6 +522,73 @@ function initialize_glm_object()
     return fit(GeneralizedLinearModel, x, y, d(), l)
 end
 
+function mle_for_θ(y::AbstractVector, μ::AbstractVector; 
```

Comment:
We are mixing notation.... theta and r is the same thing. We should just pick one